modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -193,5 +193,10 @@ export SPACEWALK_USER='<adminname>';export SPACEWALK_PASS='<password>'
 ----
 ln -s /usr/local/bin/cent-errata.sh /etc/cron.daily
 ----
+.Add the following line to you rhn.conf:
++
+----
+java.allow_adding_patches_via_api = centos7-x86_64-updates,centos7-x86_64,centos7-x86_64-extras
```

Comment:
Can this be genericised for the CentOS version?

Reply:
i think for 6 and 7 but 8 might be a bit different. In any case these labels need to match the ones from --include-channels in the script from the previous script.

Reply:
Right. Let's leave it as-is for now. We'll need to update it to 8 at some point, though.